symfony/src/Service/DeceasedNotificationService.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Service;
+
+use App\Repository\SiteRepository;
+use App\Repository\DeceasedLogRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\Persistence\ManagerRegistry;
+use Pmi\Audit\Log;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use Twig\Environment;
+
+class DeceasedNotificationService extends EmailNotificationService
+{
+    protected $em;
+    protected $managerRegistry;
+    protected $participantSummaryService;
+    protected $loggerService;
+    protected $env;
+    protected $params;
+    protected $twig;
+    protected $siteRepository;
+    protected $logRepository;
+    protected $type = 'deceased';
+    protected $render;
+    protected $time = 'deceasedAuthored';
+    protected $log;
+    protected $statusText;
+    protected $status = 'deceasedStatus';
+    protected $deceasedStatus;
+
+    public function __construct(
+        EntityManagerInterface $em,
+        ManagerRegistry $managerRegistry,
+        ParticipantSummaryService $participantSummaryService,
+        LoggerService $loggerService,
+        EnvironmentService $env,
+        ParameterBagInterface $params,
+        Environment $twig,
+        SiteRepository $siteRepository,
+        DeceasedLogRepository $deceasedLogRepository
+    ) {
```

Comment:
Can we reduce the number of injected dependencies? Since you need already have the entity manager, you could avoid injecting the specific entity repositories and just use the entity manager to get the required repositories (e.g. `$this->em->getRepository(Site::class)`)

It would also be ideal to just need one of an `EntityManagerInterface` or `ManagerRegistry`. I haven't tested this, but it looks like the `ManagerRegistry` has a `getManager` method which might return the entity manager. In that case, you could just inject the manager registry and then set `$this->em = $managerRegistry->getManager()`.